src/Microsoft.VisualStudio.Editors/OptionPages/xlf/GeneralOptionPageResources.tr.xlf

```diff
@@ -19,7 +19,7 @@
       </trans-unit>
       <trans-unit id="General_FastUpToDateCheck_LogLevel_None">
         <source>None</source>
-        <target state="translated">Yok</target>
+        <target state="translated">yok</target>
```

Comment:
Converted to lower case, which looks wrong. It seems like all translations of this key have been incorrect.